device/ospfv3/routerange/v6routerange.yaml

```diff
@@ -0,0 +1,100 @@
+components:
+  schemas:
+    Ospfv3.V6RouteRange:
+      description: >-
+        Emulated OSPFv3 IPv6 routes.
+      type: object
+      required: [name]
+      properties:
+        name:
+          x-include: ../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        addresses:
+          description: >-
+            A list of group of IPv6 route addresses.
+          type: array
+          items:
+            $ref: '../routes/routeaddresses.yaml#/components/schemas/V6RouteAddress'
+          x-field-uid: 2
+        metric:
+          description: >-
+            The user-defined metric associated with this route range.
+          type: integer
+          format: uint32
+          default: 0
+          maximum: 16777215
+          x-field-uid: 3
+        route_origin:
+          description: >-
+            The type of the OSPFv3 routes.
+          $ref: '#/components/schemas/Ospfv3V6RR.RouteOrigin'
+          x-field-uid: 4
+    
+    Ospfv3V6RR.RouteOrigin:
+      description: |-
+        Container of type of the OSPFv3 types correspond directly to the OSPFv3 LSAs types.  
+      type: object
+      properties:
+        choice:
+          description: >- 
+            Supported types are:
+            - intra_area: for Intra-Area.
+            - inter_area: for Inter Area.
+            - external_type_1: for Autonomous System (AS) External with internal AS meteric.
+            - external_type_2: for Autonomous System (AS) External with internal and external AS meteric.
+            - nssa_external: for type 7 Not-So-Stubby Area (NSSA) External.
+          type: string
+          default: inter_area
+          x-field-uid: 1
+          x-enum:
+            intra_area:
+              x-field-uid: 1
+            inter_area:
+              x-field-uid: 2
+            external_type_1:
+              x-field-uid: 3
+            external_type_2:
+              x-field-uid: 4
+            nssa_external:
```

Comment:
Might be good to clarify how to setup NSSA areas , probably need non zero area id + NSSA capability +  external type routes should not be sent in the positive test case .. maybe need a warning from implementation etc .  Basically thinking about impact of multiple interfaces in the model, some say area 0 and some area 1 with or without nssa , user will probably not be able to emulate sending specific routes for specific interfaces depending on type .. for now with single interface it is fine but might not scale well with true multiple interfaces. Need some tests to work out correct model for this. Might need to push routes to interface OR clarify in routes that either routes will be converted to NSSA or External if type mismatches or NSSA interfaces will not transmit External routes etc.

Reply:
This might require documentation changes (in model) based on findings in test. Non-blocking for initial merge.